system/prometheus-vmware/templates/_prometheus.alerts.tpl

```diff
@@ -0,0 +1,181 @@
+groups:
```

Comment:
You'll get these alerts for free as they are already part of the [common/prometheus-server Helm Chart](https://github.com/sapcc/helm-charts/blob/master/common/prometheus-server/templates/alerts/_prometheus.alerts.tpl).